projects/bp-gallery/src/components/routes.tsx

```diff
@@ -74,6 +77,30 @@ const routes: RouteConfig[] = [
       return <PictureView initialPictureId={match.params.id} />;
     },
   },
+  {
+    path: '/exhibitiontool/:id',
+    render: ({ match }: RouteConfigComponentProps<{ id: '' }>) => {
+      return <ExhibitionTool exhibitionId={match.params.id} />;
+    },
+  },
+  {
+    path: '/exhibitionOverview/:id',
+    render: ({ match }: RouteConfigComponentProps<{ id: '' }>) => {
+      return <ExhibitionFullOverview archiveId={match.params.id} />;
+    },
+  },
+  {
+    path: '/exhibitionOverview',
+    render: () => {
+      return <ExhibitionFullOverview archiveId={undefined} />;
+    },
+  },
```

Comment:
Maybe copress this down to: 
```suggestion
  {
    path: '/exhibitionOverview/:id?',
    render: ({ match }: RouteConfigComponentProps<{ id: '' }>) => {
      return <ExhibitionFullOverview archiveId={match.params.id} />;
    },
  },
```

Reply:
thank you!